pkg/cli/cvd.go

```diff
@@ -106,6 +115,21 @@ func (c *cvdCreator) createCVDFromLocalBuild() ([]*hoapi.CVD, error) {
 	if err != nil {
 		return nil, fmt.Errorf("Error building list of required image files: %w", err)
 	}
+	tarName, err := createCVDHostPackageTar(filepath.Join(vars.HostOut, CVDHostPackageDirName))
```

Comment:
The cvd-host_package.tar.gz is not supposed to include everything under $ANDROID_HOST_OUT. It would be best to get the tar file created by the build system and ask the user to run `m hosttar` if it doesn't exist.

Reply:
good call, done.